src/new-client/src/Web/MainRoute.tsx

```diff
@@ -36,19 +44,35 @@ const MainWrapper = styled.div`
   overflow: hidden;
 `
 
-export const MainRoute: React.FC = () => (
-  <Wrapper>
-    <DisconnectionOverlay />
-    <AppLayoutRoot data-qa="app-layout__root">
-      <Header />
-      <MainWrapper>
-        <Resizer defaultHeight={30}>
-          <LiveRates />
-          <Trades />
-        </Resizer>
-        <Analytics />
-      </MainWrapper>
-      <Footer />
-    </AppLayoutRoot>
-  </Wrapper>
-)
+export const MainRoute: React.FC = () => {
+  const tearOutTileState = useTearOutSectionState$("liverates")
+  const tearOutAnalyticsState = useTearOutSectionState$("analytics")
+  const tearOutTradeState = useTearOutSectionState$("trades")
+  const tearOutEntry = useTearOutSectionEntry()
```

Comment:
This hook gives us access to all the data w need.
Rather than creating 4 subscriptions in total, we could use this and maintain the tearOut state inside this component.
In other words - listen to the `tearOutEntry` and add/remove the section from some local state, and use that to conditionally render the sections

Reply:
Deleted the other three hooks, now we have a single state, I don't know if we prefer 3 different states for each section or a single state with all 3 keys (the latter makes updating the state simplier at first glance):

`
...
const [tornOutSectionState, setTornOutSectionState] = useState({
    liverates: false,
    trades: false,
    analytics: false,
  })

  useEffect(() => {
    if (tearOutEntry) {
      const [tornOut, section] = tearOutEntry
      setTornOutSectionState({ ...tornOutSectionState, [section]: tornOut })
      ...
`